src/dna.js

```diff
@@ -0,0 +1,116 @@
+const DNA_DELIMITER = "-";
```

Comment:
all DNA related code has been moved into this module for testability and organisation reasons, no code changes